README.md

```diff
@@ -96,6 +96,8 @@ Edit the default configuration of the server by adding options to your **laravel
 | `databaseConfig` |  `{}`    |  Configurations for the different database drivers [Example](#database)|
 | `hostname`     | `http://localhost` | The host of the socket.io server ex.`app.dev` |
 | `port`         | `6001`         | The port that the socket.io server should run on |
+| `protocol`     | `http`         | either `http` or `https` |
+| `referrers`     | `[]`         | ex.`['abc.com','xyz.com']` check [Referrers](#referrers) |
```

Comment:
This property is an array of objects, not an array of strings like `['abc.com','xyz.com']`. I would just point to the referrers section since an API key for that referrer needs to be generated.